src/Workspaces/Remote/ServiceHub/Services/SourceGeneration/RemoteSourceGenerationService.cs

```diff
@@ -143,10 +143,25 @@ public ValueTask<ImmutableArray<SourceGeneratorIdentity>> GetSourceGeneratorIden
         {
             var project = solution.GetRequiredProject(projectId);
             var analyzerReference = project.AnalyzerReferences
-                .OfType<AnalyzerFileReference>()
```

Comment:
this was a mistake.  And will not work once https://github.com/dotnet/roslyn/pull/74780 goes in.  In that PR we no longer have AnalyzerFileReference as the outermost type in the OOP side.  Instead, we have a wrapper AnalyzerReference type that ensures that we track if analyzers/generators are asked for.